### PR TITLE
Settings page

### PR DESCRIPTION
**Pull request by sjramirez**
Initial HTML and CSS for the settings page

### CODE REVIEW

File: out/CI_Tester.js_sum_difference.js.html [hunk 1]
```diff
@@ -45,8 +45,10 @@ <h1 class="page-title">Source: CI_Tester.js/sum_difference.js</h1>
  function mult(a, b) {
   return a * b;
 }
-
-module.exports = sum;</code></pre>
+//Want to test codacy
+alert("HELLO");
```
Comment (dcao) [inline out/CI_Tester.js_sum_difference.js.html:49]: Should this be removed

**Review by dcao — APPROVED** [review]
Looks good except for comment - fix and merge.

**Comment by dingjuan** [comment]
Probably need to resolve the conflicts before merging into the main branch

File: out/CI_Tester.js_sum_difference.js.html (hunk 1 above)
Comment (sjramirez) [reply out/CI_Tester.js_sum_difference.js.html:49]: I dont know what that is I didnt touch any of those files.

Comment (sjramirez) [reply out/CI_Tester.js_sum_difference.js.html:49]: I think it is an output from whatever they were doing with the ci

**Comment by sjramirez** [comment]
merged main to fix the auxiliary files that had conflicts

**Review by JasonSatherr — APPROVED** [review]
Looks nice

**Pull request merged**